src/generate_value.rs

```diff
@@ -0,0 +1,5 @@
+use rand::Rng;
+
+pub fn get_value() -> u8 {   
+   return rand::thread_rng().gen_range(0..100);
+}
```

Comment:
No rust, pelo que vi, podemos retornar o valor sendo o último item do escopo atual, sem usar a palavra return. Mas, por algum motivo, ele vai precisar ficar sem ;

```suggestion
pub fn get_value() -> u8 {   
   rand::thread_rng().gen_range(0..100)
}
```

Reply:
Um lembrete (pq eu esqueci quando tava fazendo), acho que quando não usar a palavra return, você obrigatóriamente precisa tipar o retorno da função/método.

Reply:
curiosidade: o Ruby segue a mesma ideia de retorno por final de escopo.